### ISSUE TITLE
Unable to run rake db:migrate

### ISSUE THREAD

**Issue by c0mrade** [jruby/warbler#307]
Hello,

This is my first time using warbler, I hope I'm overlooking something, however after few hours still unable to find the solution, here is what I mean.

Placement of warbler in Gemfile :

```
platforms :jruby do
  gem 'jruby-jars', '1.7.11'
  gem 'jruby-openssl', :require => false
  gem 'activerecord-jdbcpostgresql-adapter'
  gem 'rubyzip'

  group :development do
    gem 'jruby-lint'
    gem 'warbler', "~> 1.4.0"
  end
end
```

This is my config warble.rb :

```
Warbler::Config.new do |config|
  config.features = %w(runnable)
  config.bundle_without = %w(development test assets veracode)
  config.jar_name = "ROOT"
  config.override_gem_home = false
  config.webxml.jruby.min.runtimes = 1
  config.webxml.jruby.max.runtimes = 1
end
```

So when I build and deploy my war file, successfully might I add. I'm unable to execute `rake db:migrate`, this is what I tried :

```
java -jar /apps/tomcat/webapps/ROOT.war -S db:migrate
java -jar /apps/tomcat/webapps/ROOT.war -S db:rollback
```

This is the error I get :

```
rake aborted!
LoadError: no such file to load -- warbler
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:240:in `load_dependency'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
/private/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/lib/tasks/deployment.rake:2:in `(root)'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:1:in `(root)'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:240:in `load_dependency'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:268:in `load'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:268:in `load'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/railties-4.2.0/lib/rails/engine.rb:658:in `run_tasks_blocks'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/railties-4.2.0/lib/rails/engine.rb:658:in `run_tasks_blocks'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/railties-4.2.0/lib/rails/application.rb:438:in `run_tasks_blocks'
/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/gems/gems/railties-4.2.0/lib/rails/engine.rb:453:in `load_tasks'
/private/var/folders/qf/0kv_t7ld097_h5nc0ppfvsz40000gn/T/jruby177811857457297588extract/Rakefile:6:in `(root)'
(See full trace by running task with --trace)
```

However if I try this, works like a charm :

```
java -jar /apps/tomcat/webapps/ROOT.war -S rails c
```

What else can I do/try?

**Comment by c0mrade** [comment]
After some time trying other stuff I solved the issue.

I've got a rake task deployment.rake which builds war inside by first requiring the warble require 'warbler', and then invoking the war warble task, just like described in the README.

However since in my Gemfile, warbler is only available in development group, that was what cause the issue.

So I did this :

```
gem 'warbler', "~> 1.4.0"

platforms :jruby do
  gem 'jruby-jars', '1.7.11'
  gem 'jruby-openssl', :require => false
  gem 'activerecord-jdbcpostgresql-adapter'
  gem 'rubyzip'

  group :development do
    gem 'jruby-lint'
  end
end
```

Solved the issue
